fozzie-config/.eslintrc

```diff
@@ -1,17 +1,9 @@
 {
     "extends": "@justeat/eslint-config-fozzie",
 
-    "plugins": [
-        "jest"
-    ],
-
-    "rules": {
-        "jest/no-disabled-tests": "warn",
-        "jest/no-focused-tests": "error",
-        "jest/no-identical-title": "error"
-    },
-
     "env": {
-        "jest/globals": true
+        "node": true,
```

Comment:
These are included as part of the latest `eslint-config-fozzie` – should be able to get rid of them I’d have thought